src/flow.c

```diff
@@ -266,6 +266,29 @@ static inline int FlowUpdateSeenFlag(const Packet *p)
     return 1;
 }
 
+static inline void FlowUpdateTTL(Flow *f, Packet *p, uint8_t ttl)
+{
+    if (FlowGetPacketDirection(f, p) == TOSERVER) {
+        if (ttl < f->min_ttl_toserver) {
+            f->min_ttl_toserver = ttl;
+        } else if (f->min_ttl_toserver == 0) {
+            f->min_ttl_toserver = ttl;
+        }
+        if (ttl > f->max_ttl_toserver) {
+            f->max_ttl_toserver = ttl;
+        }
```

Comment:
How about compressing this to:
<pre>
if (f->min_ttl_toserver == 0) {
    f->min_ttl_toserver = ttl;
} else {
   f->min_ttl_toserver = MIN(f->min_ttl_toserver, ttl);
}
f->max_ttl_toserver = MAX(f->max_ttl_toserver, ttl);
</pre>

Reply:
OK and same I suppose for the next piece of code (toclient).